### PR TITLE
Added leader election and k8s events documentation

### PR DESCRIPTION
**Pull request by orhanasan**

### CODE REVIEW

File: configuration/inputs.md [hunk 1]
```diff
@@ -233,10 +234,19 @@ A saved script can also be referenced directly via the command parameter \(see s
...
         for i in {1..50}
           do
-            echo "Welcome $i times" 
+            echo "Welcome $i times"
           done
 ```
 
+## Kubernetes Events
+
+If enabled, agents will report the warning-level events generated by Kubernetes (namespace independent). Note that Kubernetes Events should be used with `ED_LEADER_ELECTION_ENABLED` environment variable in order to observe events.
```
Comment (taylanisikdemir) [inline configuration/inputs.md]: please replace "agents will report the warning-level events" with "agents will collect events"

no need to mention about our Warning filter that is applied in the API level. we actually collect all events

Comment (taylanisikdemir) [reply configuration/inputs.md]: Let's also mention about the env variable's value
`ED_LEADER_ELECTION_ENABLED=1`

File: installation/environment-variables.md [hunk 2]
```diff
@@ -36,9 +36,16 @@ description: >-
...
         according their prefixes and disable the rest together with ED_WORKFLOWS</td>
-      <td
-      style="text-align:left">prefix:prefix:...</td>
-        <td style="text-align:left">workflow_prod_:workflow_cache_</td>
+      <td style="text-align:left">prefix:prefix:...</td>
+      <td style="text-align:left">workflow_prod_:workflow_cache_</td>
+    </tr>
+    <tr>
+      <td style="text-align:left">ED_LEADER_ELECTION_ENABLED</td>
+      <td style="text-align:left">Should be set to "1" in order Kubernetes leader election mechanism to be enabled.
```
Comment (taylanisikdemir) [inline installation/environment-variables.md]: ".. in order to enable agent leader election mechanism in Kubernetes environment. ..."

**Review by taylanisikdemir — APPROVED** [review]
thanks for adding this. let's hold onto this PR until we release v0.1.8 which has some fixes on k8s event tailing.

File: installation/environment-variables.md (hunk 2 above)
Comment (orhanasan) [reply installation/environment-variables.md]: Changed it accordingly

File: configuration/inputs.md (hunk 1 above)
Comment (orhanasan) [reply configuration/inputs.md]: Done

**Comment by taylanisikdemir** [comment]
Merging this since we released v0.1.8 which has k8s event tailing

**Pull request merged**